release/models/qos/openconfig-qos-interfaces.yang

```diff
@@ -377,6 +377,30 @@ submodule openconfig-qos-interfaces {
         "Number of octets dropped by the queue due to overrun, that is tail-drop or WRED drop";
     }
 
+    leaf ecn-selected-pkts {
+      type oc-yang:counter64;
+      description
+        "Number of packets enqueued while AQM algoritm selects the packet.
+        
+        For RED/WRED AQMs it would be when queue utilization is:
+          - greather then max-thershold, or
+          - between min-threshold and max-threshold, and with probability as determinaed with WRED/RED slope.
+        
+        The packet counteed regardless of its original ECN codepoint.";
+    }
+
+    leaf ecn-selected-octets {
+      type oc-yang:counter64;
+      description
+        "Number of octets enqueued while AQM algoritm selects the packet.
+        
+        For RED/WRED AQMs it would be when queue utilization is:
+          - greather then max-thershold, or
+          - between min-threshold and max-threshold, and with probability as determinaed with WRED/RED slope.
+        
+        The packet's octed are counted regardless of its original ECN codepoint.";
+    }
+
```

Comment:
Given the implementation references, it seems only Cisco implements counters that differentiate 'selected' and 'marked' at this time.  Is there any indication of support for both selected and marked from other implementations?

Reply:
No cisco implemets **only** `marked`.
I've put both, as industry-wise both has sens. Implementation may suport either or (preferebly) both. 
Since this is ASIC/NPU dependent, it alows flexibility.

`selected' provides clear and unbiaed signal that five queue is occupied to the point it will remark ECT -> CE if such traffic arrive. And do this even if at the time all incoming traffic is CE11 / non-ECN-capable. 

`marked` provides clear signal about packet that are erived as ECT and wer remarked to CE. It is bit harder for me to find use of it. If 'selected' is not available, this could be some substitution but biased. If all incoming traffic is CE or non-ECN, counter would not increase, regardless of queue utilization.

Reply:
I see, some implement selected, some implement marked, nobody implements both.  We need both defined to capture the behavior accurately.  Makes sense to me.

Can you update the [PR description](https://github.com/openconfig/public/pull/935#issue-1827185832) to clearly indicate your interpretation of selected vs. marked for each referenced implementation?